README.md

```diff
@@ -891,6 +891,30 @@ modifications necessary).
 Note that it will not migrate functions and triggers, since `migrate`
 is `FALSE` by default for these objects.
 
+Detailed description of the statements functions
```

Comment:
I am not happy with that heading.

Perhaps we could call it

```none
Low-level migration functions
```

`materialize_foreign_table` and `execute_statement` should also be documented in this section.

The documentation could need some more love. In particular, you should document that it returns a table, and describe the use case.

Reply:
[f3ea21e](https://github.com/cybertec-postgresql/db_migrator/pull/33/commits/f3ea21e5113b91c74959c3be673fed6aa79e5c21) brings what you suggest. I agree with you and I will take an extra effort to reflect all those new functions in a detailed manner in this section!